stable/kubernetes-dashboard/templates/svc.yaml

```diff
@@ -11,15 +11,24 @@ metadata:
 {{- if .Values.service.labels }}
 {{ toYaml .Values.service.labels | indent 4 }}
 {{- end }}
+{{- if .Values.ingress.enabled }}
+  annotations:
+    service.alpha.kubernetes.io/app-protocols: '{"https":"HTTPS"}'
```

Comment:
Do we need this ?
We can configure this in annotations

Reply:
It is required to make it work with GCE ingress yes. But I'm not sure I understand what you mean. It is an annotation and configured as such but did you mean via values instead of in the template? if so It would have to be clear that we need that to make it work with GCE ingress. And it is actually to match that: https://github.com/helm/charts/pull/9179/files#diff-87590f01f77a035df78b589038b308d7R31

I had a hard time figuring that out.

Reply:
I'm not sure how this configuration will work on other service providers

Reply:
Yeah I'm not in a position to test that honestly. If we would make that optional, how is the best approach doing this given it took me a good amount of time to make it work and given I want to save time to other people :). Also google is deprecating the dashboard addon part of GKE so anyone who wants it working on GKE will have that problem.

Reply:
Ok I update the PR @maver1ck. I can do what I want with values except one thing which is the actual fix.